.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: Testing Interoperability
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on: workflow_dispatch
+jobs:
+  Testing_Interoperability:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Downloads assets
+        uses: robinraju/release-downloader@v1.7
+        with:
+          latest: true
+          fileName: "*"
+      - name: Unzip
+        uses: montudor/action-zip@v1
```

Comment:
[Optional] You can remove the usage of this action and make the pipeline simpler. You can replace this step by:
```
- name: unzip
  run: tar -xvf ...
```
You can even use unzip, check the installed tools list [here](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md)

Reply:
okey, i changed it to unzip '*.zip' -d executables